woocommerce/woocommerce-android
# Fix swipe to refresh in analytics hub section

Closes:  [5426](https://github.com/woocommerce/woocommerce-android/issues/5426)

### Description
This pull request adds scrollable view to analytics fragment and implements pull to refresh action

### Testing instructions

1. Open the application compiled in debug mode
2. Click on the analytics section in the bottom bar navigation
3. Do a swipe to refresh action in the section
4. Data should be reloaded

### Images/gif
<img src="https://user-images.githubusercontent.com/572601/145399015-25ded608-7811-438c-93ac-62202b375f41.gif" width="280" height="600">
